lib/api/controllers/authController.js

```diff
@@ -75,7 +75,7 @@ function AuthController (kuzzle) {
    * @returns {Promise}
    */
   this.getCurrentUser = function (requestObject, context) {
-    requestObject.data._id = context.token.user;
+    requestObject.data.body._id = context.token.userId;
```

Comment:
Seems weird that we need to put the user id in the body part of the request object, when this field is directly in the data part for every other API requests. Can you take a look at it?